vrk-kpa/suomidigi
# SUOPA-771 Sub navigation theming

To test:
- `make fresh`
- Log in and go to https://suomidigi.docker.sh/admin/structure/menu/manage/main
    - Create enough menu links to get a tree-like link collection (like below). Make sure to add `/node/<nid>` for each link to gain access to the actual page. 
![image](https://user-images.githubusercontent.com/1712902/71578202-d4984f00-2aff-11ea-8647-7a7d0028813c.png)
- Go to one of the node you've added to the links and check that the sidebar menu looks some what like this: 
![image](https://user-images.githubusercontent.com/1712902/71578327-5e481c80-2b00-11ea-86f3-041ef5a6d5e2.png)
- Also check that the mobile version works as well.
